tests/test_django/urls.py

```
@@ -3,70 +3,255 @@
+        #         "authors__name",
+        #         "publisher__name",
+        #         "publication_date",
+        #     ]
+    )
 
-    class Meta(Crudl.Meta):
-        """Configuration for the CRUDL API."""
 
-        model_class = models.Publisher
-        base_path = "/publishers"
+class LibraryCrudl(CrudlController[Library]):
```

Ruff (0.8.2)

Missing docstring in public class

(D101)